ci/prs.yml

```diff
@@ -1,14 +1,19 @@
 # Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-trigger: none
-
-pr:
-  autoCancel: true # cancel previous builds on push
+#trigger: none
+trigger:
   branches:
-    include:
-      - main
-      - release/*
+    exclude:
+    - main
+    - release/*
+
+pr: none
```

Comment:
how does this interact with checks on github for PRs? Do they still show up (I expect so, iirc that’s what happened for manually triggered builds for a branch) and are they still blocking (not sure, at least for manual builds I think that was not the case)?

Reply:
If I understand this correctly, this effectively means that we’re giving up on not being able to schedule builds for arbitrary commits without leading some form of audit trail in the form of a PR which has security implications. Not necessarily saying this is a bad idea but probably at least worth thinking about.

Reply:
> how does this interact with checks on github for PRs? Do they still show up and are they still blocking?

Yes and yes. The fact that they are blocking has nothing to do with why they started, it's just based on the name of the status report. Manual runs are blocking right now if they're runs of the PR pipeline (which we very rarely do for obvious reasons); manual runs of the compat pipeline are not blocking because the compat pipeline is not blocking.